test/test_api_docs/swagger_template.py

```diff
@@ -0,0 +1 @@
+SWAGGER_TEMPLATE = {'openapi': '3.0.1', 'info': {'title': 'TestApp', 'description': 'Running on: <strong>ServerName</strong> on: <strong>[]</strong><p>This is a test app for TestDocWebHandler.</p>', 'contact': {'name': 'ASAB microservice', 'url': 'https://www.github.com/teskalabs/asab'}, 'version': {}}, 'servers': [{'url': '../../'}], 'paths': {'path/to/file': {'put': {'summary': 'This is a test handler.', 'description': 'This is a test handler.\n\nReturns:\n        nothing special\n\nHandler: `MockedRouterObject.handler()`', 'tags': ['test_handler'], 'responses': {'200': {'description': 'Success'}}}, 'get': {'summary': 'This is a test handler.', 'description': 'This is a test handler.\n\nReturns:\n        nothing special\n\nHandler: `MockedRouterObject.handler()`', 'tags': ['test_handler'], 'responses': {'200': {'description': 'Success'}}}}, 'another/path': {'get': {'summary': 'This is a test handler.', 'description': 'This is a test handler.\n\nReturns:\n        nothing special\n\nHandler: `MockedRouterObject.handler()`', 'tags': ['test_handler'], 'responses': {'200': {'description': 'Success'}}}}}, 'components': {'securitySchemes': {}}}
```

Comment:
the `SWAGGER_TEMPLATE` is not a template, i'd call it something like `TEST_ALL_KEYS_EXPECTED_OUTPUT`.

and since you have it in a separate file, you can call the file `expected_output.py`